pkg/azureclients/fileclient/azure_fileclient.go

```diff
@@ -200,3 +200,74 @@ func (c *Client) GetServiceProperties(ctx context.Context, resourceGroupName, ac
 func (c *Client) SetServiceProperties(ctx context.Context, resourceGroupName, accountName string, parameters storage.FileServiceProperties) (storage.FileServiceProperties, error) {
 	return c.fileServicesClient.SetServiceProperties(ctx, resourceGroupName, accountName, parameters)
 }
+
+// CreateSnapshot creates a snapshot
+func (c *Client) CreateSnapshot(ctx context.Context, resourceGroupName, accountName string, shareOptions *ShareOptions) error {
```

Comment:
Could you add some unit tests for those new functions?

Reply:
We use old function CreateFileShare to support snapshot later.